Neural Net 2/Population.cpp

```diff
@@ -1,3 +1,18 @@
 #include "stdafx.h"
 
 #include "Population.h"
+
+Population::Population(int *structure, int size)
+{
+	this->population_size = size;
+	this->individus = new Red[size];
+
+	for (int i = 0; i < this->population_size; i++)
+	{
+		Red red = Red(structure);
```

Comment:
Est il nécessaire de créer une nouvelle Red avec les paramétrés et après copier ce réseau sur `this->individus[i]`?
Peut-on le faire directement sur `individus[i]` ?

Reply:
Oui, je crois que tu as raison ^^